Map Version Edits.R

```diff
@@ -71,21 +92,18 @@ network <- suppressWarnings({
 })
 
 
-# Assuming 'network' is your sfnetwork
 network_sf <- st_as_sf(network)
 
-
-ggplot() +
-  geom_sf(data = st_as_sf(town_osm_center), aes(col = highway))
```

Comment:
I'm assuming this plot was just used for debugging purposes, but let me know if you think it should be a necessary part of running the script!

Reply:
It was for debugging.